src/export/export.js

```diff
@@ -0,0 +1,487 @@
+import React, {useContext, useEffect, useState} from 'react';
+import {
+  View,
+  Text,
+  TouchableOpacity,
+  StyleSheet,
+  SafeAreaView,
+  TextInput,
+  Alert,
+} from 'react-native';
+import {getArrayOfDates} from '../services/date/helpers';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import ExportData from '../../assets/svg/export-data.svg';
+import {colors} from '../common/colors';
+import {DiaryDataContext} from '../context';
+import {displayedCategories} from '../common/constants';
+
+const MailStorageKey = '@Mail';
+
+//TODO : identité du sender de mail
+const mailFrom = '';
+const mailFromName = 'MonSuiviPsy - Application';
+const API_USER = '2fbfeec4905f352f871b2590da840571';
+const API_KEY = 'e94f70b1c2dd423a446efbbc788200cb';
+
+const Export = ({navigation}) => {
+  const [mail, setMail] = useState('');
+  const [loading, setLoading] = useState(false);
+  const [days, setDays] = useState(30);
+  const [diaryData] = useContext(DiaryDataContext);
+
+  useEffect(() => {
+    (async () => {
+      const m = await AsyncStorage.getItem(MailStorageKey);
+      if (m) setMail(m);
```

Comment:
Mettre des accolades autour des instructions des `if` pour respecter les règles Eslint
```suggestion
      if (storageMail) {
        setMail(storageMail);
      }
```